hollow/src/main/java/com/netflix/hollow/api/producer/HollowIncrementalProducer.java

```diff
@@ -76,13 +76,22 @@ public void delete(Object obj) {
     public void delete(RecordPrimaryKey key) {
         mutations.put(key, DELETE_RECORD);
     }
-    
+
+    public void clearMutations() {
```

Comment:
I like that this is public. A user might decide to abandon a set of mutations previously queued up. This allows them to do that without having to discard the producer to gain that effect.

Reply:
👍